packages/modules/src/space/Space.chain.ts

```diff
@@ -11,39 +11,42 @@ import type {
   IPublicIdentity,
   SubmittableExtrinsic,
 } from '@cord.network/types'
-import { SPACE_PREFIX } from '@cord.network/types'
 import { DecoderUtils, Identifier } from '@cord.network/utils'
 import { ConfigService } from '@cord.network/config'
 import { ChainApiConnection } from '@cord.network/network'
 import { Identity } from '../identity/Identity.js'
 
-const log = ConfigService.LoggingFactory.getLogger('Schema')
+const log = ConfigService.LoggingFactory.getLogger('Registry')
```

Comment:
is this intentional to call this 'Registry'? or should we stick with the 'Space'?